ethereum/contracts/pyth/PythGovernanceInstructions.sol

```diff
@@ -0,0 +1,134 @@
+// contracts/GovernanceStructs.sol
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "../libraries/external/BytesLib.sol";
+import "./PythInternalStructs.sol";
+
+/**
+ * @dev `PythGovernanceInstructions` defines a set of structs and parsing functions
+ * for Pyth governance instructions.
+ */
+contract PythGovernanceInstructions {
+    // module 2 corresponds to Target Chain contracts
+    uint8 constant MODULE = 2;
+
+    using BytesLib for bytes;
+
+    struct GovernanceInstruction {
+        uint8 module;
+        uint8 action;
```

Comment:
Could this field have the `GovernanceAction` type, and do the mapping from `uint` to `GovernanceAction` at deserialization time? This would make it less likely to make a mistake later in the code.